docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -5,7 +5,7 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 = Building a Native Executable
 include::_attributes.adoc[]
-:categories: getting-started
+:categories: getting-started, native
```

Comment:
Not sure if it matters, but all other docs have no space between items, just a comma, like so:

```suggestion
:categories: getting-started,native
```

Reply:
whitespace between items doesn't matter (but the comma does. ;) )

Reply:
Thanks @ebullient